### ISSUE TITLE
Define teams for `docsy-maintainers`, `docsy-approvers`, and the steering committee

### ISSUE THREAD

**Issue by chalin** [docsy/docsy#1735]
As a step towards formalizing Docsy team roles (and their associated responsibilities and access rights), I'd like to suggest that we create the following GH teams:

- Docsy maintainers,`docsy-maintainers`
- Docsy approvers, which includes maintainers as a team member - `docsy-approvers`
- Docsy steering committee:
  - I think that we might want to adopt the term "technical steering committee" (TSC) rather than "project steering committee" (PSC) as introduced in the [Hello Docsy!](https://www.docsy.dev/blog/2022/hello/#introducing-the-psc) blog. Just a thought.
  - Might "governance committee" be better?
  - Depending on the answers to the questions above, we'd have possibly, one of the following. I'm trying to find a short but meaningful name (even if it isn't PSC or TSC):
    -  `docsy-steering` feels like there's a word missing; `docsy-steerers` (wow, I didn't think that was a word, but it is!); or maybe `docsy-leads`
    -  `docsy-steering-committee`; or the shorter `docsy-sc` but that seems a bit cryptic; as does `docsy-tsc`
    -  `docsy-governance-committee`; or `docsy-gc`, which feels a little less cryptic to me, maybe because I'm used to seeing that abbreviation used at the CNCF?

AFAIK, teams need to be defined at the org level. Can you look into this @emckean?

**Edit**: waiting on

- #1992

**Comment by chalin** [comment]
It will also allow us to set CODEOWNERS.

**Comment by emckean** [comment]
These teams have been created; can you see them under Collaborators and Teams?

**Comment by chalin** [comment]
No, I can't see the teams and they don't seem to be accessible from here -- that is, in issue comments. Normally, I should be able to use `@` completion to find teams, but that isn't showing any teams starting with `docsy`.

**Comment by emckean** [comment]
testing @google/docsy-approvers
